src/common/service.rs

```diff
@@ -0,0 +1,56 @@
+use crate::core::configuration::{ServiceConfiguration};
+use crate::core::wallet::{Wallet, OwnedCrypto};
+
+pub fn bot_service(config:ServiceConfiguration) {
+  let wallet = Wallet {
+    user_alias: config.user_alias.clone()
+  };
+  let item_optional = wallet.get_holding_item(&config.crypto);
+  match item_optional {
```

Comment:
```suggestion
  let item_optional = wallet.get_holding_item(&config.crypto);

  match item_optional {
```

Please add a blank line after a declaration block.